File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.cc
#include <iostream>
using namespace std;

void CheckP(int* plist, int* piece) {
    for (int i = 0; i < 6; i++) {
        piece[i] = piece[i] - plist[i];
    }
}

int main(void) {
    ios::sync_with_stdio(false);
    cin.tie(NULL);

    int plist[6];
    int piece[6] = { 1, 1, 2, 2, 2, 8 };

    for (int i = 0; i < 6; i++) {
        if (!(cin >> plist[i])) return 0;
    }

    CheckP(plist, piece);

    for (int i = 0; i < 6; i++) {
        cout << piece[i] << (i == 5 ? "" : " ");
    }
    cout << "\n";

    return 0;
}

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
import sys
input = sys.stdin.readline

def check_p(plist: list[int], piece: list[int]):
    result = []
    for i in range(6):
        result.append(piece[i] - plist[i])
        
    print(*result)
        
def solve():
    line = input().split()
    if not line: return
    
    plist = list(map(int, line))
    piece = [1, 1, 2, 2, 2, 8]
    
    check_p(plist, piece)
    
if __name__ == "__main__":
    solve()
